index.html

```diff
@@ -14,37 +14,45 @@ <h2>Mark</h2>
+
+        <nav>
+            <ul class="nav-content">
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+              </ul>
+        </nav>
     </header> 
 
     <section class="landing-page all-body" id="body-cover">
-               
-                <h1 class="title">I'm Mark <br>Glad to see you!</h1>
-                <p class="intro">
-                    I'm a software developer! I can help you build a product , 
-                    feature or website Look through some of my work and experience! 
-                    If you like what you see and have a project you need coded, don't 
-                    hestiate to contact me.
-                </p>
-            
-                <p class="lets-connect">LET'S CONNECT</p>
-                    
-                <ul class="social-buttons">
-                    <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
-                </ul>
+               <div class="ctn">
+                    <h1 class="title">I'm Mark <br>Glad to see you!</h1>
+                    <p class="intro">
+                        I'm a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, don't 
+                        hestiate to contact me.
+                    </p>
                 
+                    <p class="lets-connect">LET'S CONNECT</p>
+                        
+                    <ul class="social-buttons">
+                        <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
+                        <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
+                    </ul>
+               </div>
     </section>
 
 
     <section class="works-container" id="works-container">
         <div class="works">
-            <img src="images/Snapshoot Portfolio.svg" alt="" class="tonic">
-            <img src="images/Snapshoot Portfolio1.svg" alt="" class="tonic-1">
-            <div class="div1">
-                <h2 class="header-tonic">Tonic</h2>
+            <section class="illustration">
+            </section>
+            <div class="div1 w_ctn">
```

Comment:
- [OPTIONAL] There are some serious issues in your projects' cards. I will highly recommend that you fix them. these issues were missed by the previous reviewer.
   -  The heading in the card is left aligned in the figma design but in our design it is very poorly placed and dont give a good impression.
   - The  top and bottom margins of the elements in projects' description are not similar to figma design so consider adding more margin to make them identical to the figma design.